frontend/README.md

```diff
@@ -26,3 +26,25 @@ Before running the tests make sure you are serving the app via `ng serve`.
 ## Further help
 
 To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
+
```

Comment:
These explanations should be in the main README file, not in the README of the frontend.